cmake/msys/msys_helper.sh

```diff
@@ -222,9 +212,8 @@ if [ ! -e "$mingw_root/lib/libgig/libgig.dll.a" ]; then
 	make install
 
 	mv "$mingw_root/lib/bin/libakai-0.dll" "$mingw_root/bin"
-	mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"
 
-	if [ $? -ne 0 ]; then
+	if ! mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"; then
```

Comment:
I moved to `set -e` in a new commit. There's no need for custom error message IMO: we print a lot of info about what we're doing, so the user will know what failed, plus, the shell will print something too. What do you think ?